src/opening-period/time-span/TimeSpan.scss

```diff
@@ -0,0 +1,45 @@
+@import 'node_modules/hds-design-tokens/lib/all.scss';
+
+.remove-time-span-button {
+  color: var(--color-coat-of-arms);
+}
+
+.start-and-end-time-container {
+  display: flex;
+  align-items: flex-end;
+}
+
+.dash-between-start-and-end-time-container {
+  height: 56px;
+  display: flex;
+  margin-left: 10px;
+  margin-right: 10px;
+  font-family: HelsinkiGrotesk-Bold, Arial, sans-serif;
+}
+
+.time-span-state-container {
+  width: 307px;
+}
+
+.time-span-state {
+  font-family: HelsinkiGrotesk-Regular, Arial, sans-serif;
+}
+
+.time-span-description-container {
+  width: 386px;
+}
+
+.time-span-description {
+  font-family: HelsinkiGrotesk-Regular, Arial, sans-serif;
+}
+
+.time-span-first-header-row {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.time-span-container {
+  border-top: $spacing-4-xs solid var(--color-coat-of-arms);
```

Comment:
Maybe this color could be in the variables.css?

Reply:
The most fitting by name seemed to be this: `--primary-color: var(--color-coat-of-arms);` Maybe we use that?

- [ ] Apply the 'primary-color' variable found in variables.css

Reply:
Damn I think I understood the question wrong. You were suggesting that we would put that color there, not that it might already be there?

Reply:
Yes, but I guess it is ok to use this as well

Reply:
I think primary color says it